en/skills/11. Tables.ipynb

```diff
@@ -0,0 +1,458 @@
+      },
+      "source": [
+        "## Reading and Writing CSV Files\n",
+        "Just like reading from a CSV file, you can also write new data into a CSV file using Python’s `csv` module.\n",
+        "\n",
+        "#### Example (Writing):"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "e491bbfc",
+      "metadata": {
+        "id": "e491bbfc"
+      },
+      "outputs": [],
+      "source": [
+        "import csv\n",
+        "\n",
+        "data = [[\"Nawatl\", \"English\"],\n",
+        "        [\"Wehka\", \"Far\"],\n",
+        "        [\"Oksepa\", \"Again\"],\n",
+        "        [\"Xinichchia\", \"Wait for me\"]]\n",
+        "\n",
+        "with open('data.csv', 'w', newline='') as file:\n",
+        "    writer = csv.writer(file)\n",
+        "    writer.writerows(data)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Once this file is created, it will appear wherever we set the path to. Since we didn't (we just named it \"output.csv\" it's directly in the same directory where we have this same file we're currently working with."
+      ],
+      "metadata": {
+        "id": "GDSRWfJniRJQ"
+      },
+      "id": "GDSRWfJniRJQ"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "#### **Exercise 1**"
+      ],
+      "metadata": {
+        "id": "Kn6r0Z6bjF0o"
+      },
+      "id": "Kn6r0Z6bjF0o"
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "###### Show answer"
+      ],
+      "metadata": {
+        "id": "Iq7UEZjskUlZ"
+      },
+      "id": "Iq7UEZjskUlZ"
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import csv\n",
+        "\n",
+        "# Define your data\n",
+        "data = [[\"Name\", \"Age\", \"Hobbies\"],\n",
+        "        [\"John\", 28, \"Coding, Reading, Traveling\"]]\n",
+        "\n",
+        "# Write to the CSV file\n",
+        "with open('myself.csv', 'w', newline='') as file:\n",
+        "    writer = csv.writer(file)\n",
+        "    writer.writerows(data)"
+      ],
+      "metadata": {
+        "id": "BYmb-DBvkQnc"
+      },
+      "id": "BYmb-DBvkQnc",
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "##### Create a CSV file with information about yourself (Name, Age, Hobbies) and save it with the name \"myself.csv\")"
+      ],
+      "metadata": {
+        "id": "UHngqmx6kauC"
+      },
+      "id": "UHngqmx6kauC"
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Create myself.csv"
+      ],
+      "metadata": {
+        "id": "NzVPZUa9jREC"
+      },
+      "id": "NzVPZUa9jREC",
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "id": "cbd8d525",
+      "metadata": {
+        "id": "cbd8d525"
+      },
+      "source": [
+        "### Opening CSV Files with Code\n",
```

Comment:
I would do "opening csv files" before saving/exporting, since it allows you to immediately get some visual feedback

Reply:
good point, I'll change that.